### ISSUE TITLE
Animation like WhatsApp/Twitter

### ISSUE THREAD

**Issue by 26feb97shubham** [skydoves/Orbital#42]
Hello,

Good morning,

Hope you are doing well. Actually I am stuck in an issue where I have to implement the animation feature like WhatsApp and Twitter. Hope you have used Twitter and in Twitter we have a post let's say an Image Post with text and whenever we tap on the image it comes out with an animation and in the form of pop-up and if we tap again it goes to it's original position. This I have to implement for the list.
In short, whenever we click on the image it should jump from its original position to center in form of popup and on tapping back again it should come back to its original position.

Will be waiting for your revert as soon as possible.

Thank you
Shubham Jain

**Comment by skydoves** [comment]
Hey @26feb97shubham, have you ever checked out this article before?

https://medium.com/proandroiddev/shared-element-transition-in-jetpack-compose-provide-enriched-user-experiences-163d4e435869
